usmqe_tests/conftest.py

```diff
@@ -98,3 +99,13 @@ def valid_password(request):
     Return valid password string.
     """
     return request.param
+
+
+@pytest.fixture
+def os_distro():
+    """
+    Return name of os distribution.
+    """
+    SSH = usmssh.get_ssh()
+    cmd_distro = "awk -F'\"' 'NR==1{print $2}' /etc/os-release"
```

Comment:
Getting properly the OS name seems to be quite complex task, and we probably don't need to cover all possible cases, but still I would propose here to parse the /etc/os-release somehow differently, because:
* the lines order might be different
* the value might be without brackets (for example on Fedora)
* the file might be even missing (RHEL 6) - but I wouldn't consider this case

I see two possible solutions here:
* grep the file for `NAME` and parse the particular line, or
* parse whole file and return it as a dict - so it would be possible to get also for example version number (then, it would be probably better to rename the function to something like `os_info()`

Reply:
Changed the `os_distro` fixture to `os_info` as suggested. Now it contains all the data from `/etc/os-release` file.